src/3d/qgs3dmapsettings.h

```diff
@@ -595,6 +678,18 @@ class _3D_EXPORT Qgs3DMapSettings : public QObject, public QgsTemporalRangeObjec
     bool mIsSkyboxEnabled = false;  //!< Whether the skybox is enabled
     QgsSkyboxSettings mSkyboxSettings; //!< Skybox related configuration
     QgsShadowSettings mShadowSettings; //!< Shadow rendering related settings
+
+    bool mEyeDomeLightingEnabled = false;
+    double mEyeDomeLightingStrength = 1000.0;
+    int mEyeDomeLightingDistance = 1;
+
+    bool mDebugShadowMapEnabled = false;
+    QString mDebugShadowMapCorner = "Top Left";
```

Comment:
Rather than setting and returning the position as a string, it would be better to have an enumeration with the list of recognized positions - more robust to errors...

Reply:
There's a Qt enum we can reuse here -- Qt::Corner

Reply:
> There's a Qt enum we can reuse here -- Qt::Corner

Yes, I'm using that :)